src/Console/Command/DatabaseDropCommand.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Console\Command;
+
+use Patchlevel\EventSourcing\Store\DoctrineStore;
+use Patchlevel\EventSourcing\Store\Store;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class DatabaseDropCommand extends Command
+{
+    private Store $store;
+
+    public function __construct(Store $store)
+    {
+        parent::__construct();
+
+        $this->store = $store;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('event-sourcing:database:drop')
+            ->setDescription('create eventstore database');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
maybe we should here add a defensive mechanism, so it will only warn if normal executed and only execute if `--force` is added or sth. like that?

Im not sure about this just throwing this idea here out

Reply:
that was first a test so that I could try it out in my project. I have expanded it now ;-)